README.md

```diff
@@ -1,2 +1,243 @@
+Creating and working in your own feature branch allows you to commit
+and share incomplete work without breaking the project for other people.
+
+To illustrate the use of branches (which arguably aren't really needed
+for this simple project), we'll create a new branch for the creation
+of each contact info page. So in the example above, the first thing
+that Pat would do would be to create a branch for the creation of
+Chris's contact info page.
+
+In the command line this would be accomplished with
+```bash
+git checkout -b <branch_name>
+```
+so Pat might use `git checkout -b chris_info` to create the branch for
+setting up Chris's info.
+
+In WebStorm, you can accomplish this through the
+`VCS -> Git -> Branches…` dialogue, choosing the `New Branch` option.
+
+## Creating a page
+
+Once you've created the branch it's time to create your partner's
+contact info page.
+
+Each contact info page should:
+
+* Be named `<your_name>.md`, e.g., `Nic_McPhee.md`
+* Contain at least:
+   * Your name as you preferred to be called
+   * Your preferred pronouns
+   * Your preferred way(s) for people to contact you, perhaps with some
+   info on which one(s) people should use if they need to get in touch
+   quickly
+   * Any constraints on any of that info, e.g., "Don't call the
+   apartment phone after 10pm or my roommate won't do dishes for a week."
+
+Your preferred contact methods is arguably the tricky part of this.
+You should _not_ feel that you
+need to share contact info you're not comfortable sharing. So if you
+want to include a phone number, that's fine, but it's _equally_ fine if
+you don't. Whatever you do share, though, it would be good if there is
+some way that your team can contact you quickly if something blows up.
+
+:bangbang: Remember to treat everyone's information with respect and use
+it in reasonable ways. Everyone's trusting us both individually and as
+a community to not do dumb or inappropriate things with this information,
+so let's be careful to not abuse that trust.
+
+Feel free to be creative about this. Do you obsessively follow your
+Twitter account, thereby making a Twitter direct message a pretty
+reasonable approach? Also feel free to come back and update this info
+as the semester progresses. People are probably most likely to look
+things up here in the second half of the semester when the groups are
+bigger and the team coordination gets more complex, so changes you
+make are likely to still be useful even if they're several weeks after
+we "complete" this lab.
+
+## Sharing that new page
+
+After a page is created, you need to share that with the class by
+adding it to the repository on Github.
+
+Because you've created your own branch to work in, the beginning of
+this is easy:
+
+* Stage the new file so `git` knows you want to commit it
+   * `git add <filename>` on the command line
+   * `VCS -> Git -> Add` in WebStorm (although if you used WebStorm to
```

Comment:
Maybe the part about add in webstorm should be moved or also included above so that people know the answer to that question if it comes up.